src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -2265,7 +2107,7 @@ SyncMetadataToNodesMain(Datum main_arg)
 		{
 			UseCoordinatedTransaction();
 
-			MetadataSyncResult result = SyncMetadataToNodes();
+			MetadataSyncResult result = SyncNodeMetadataToNodes();
```

Comment:
we should probably rename the enum `MetadataSyncResult` to `NodeMetadataSyncResult` to have a more unified convention